spec/features/collection_drafts/progress_circles_spec.rb

```diff
@@ -7,10 +7,9 @@
     before do
       login
 
-      visit '/manage_metadata'
-      choose 'New Collection Record'
-      click_on 'Create Record'
-      
+      visit '/manage_collections'
```

Comment:
I think we should be using `visit manage_collections_path` instead of `visit '/manage_collections'` if possible